app/screens/user_profile/user_profile.js

```diff
@@ -47,17 +46,20 @@ class UserProfile extends PureComponent {
     }
 
     close = () => {
-        const {navigator} = this.props;
-
-        navigator.popToRoot({
-            animated: true
+        const {navigator, theme} = this.props;
+
+        navigator.resetTo({
```

Comment:
Why not use `popToRoot` here?

Reply:
`popToRoot` does not work well if you have one or multiple modals opened.

you can:
* Open search (modal)
* tap on a post (second modal)
* tap on a post (push to thread view)
* tap on a user (push to user profile view)
* Send message to user

Reply:
What happens with `popToRoot` in that case that's different from `resetTo`? It sounds like they should work the same by closing all the modals and screens to get back to the channel.